spec/03-plugins/17-jwt/03-access_spec.lua

```diff
@@ -254,6 +260,39 @@ for _, strategy in helpers.each_strategy() do
         local body = assert.res_status(401, res)
         assert.equal([[{"message":"Unauthorized"}]], body)
       end)
+      it("returns 403 if the token exceeds the maximum_expiration limit", function()
+        local payload = {
+	  iss = jwt_secret.key,
+	  exp = os.time() + 3600,
+	  nbf = os.time() - 30
```

Comment:
style: indentation is off here and in several other places in this file

Reply:
fixed